assets/javascripts/kitten/components/modals/modal/index.js

```diff
@@ -86,6 +87,9 @@ export class Modal extends Component {
           ariaHideApp={false}
           onRequestClose={this.close}
           contentLabel={label}
+          bodyOpenClassName={
+            disableOutsideScroll ? 'k-Modal__body--open' : null
```

Comment:
Je pense qu'on pourrait l'écrire mais j'ai un doute vu que ca va retourner `false` le cas échéant.
```suggestion
            disableOutsideScroll && 'k-Modal__body--open'
```

Reply:
je teste

Reply:
ça marche